.travis.yml

```diff
@@ -13,24 +13,20 @@ cache:
     - $HOME/.gradle/daemon
     - $HOME/.android/build-cache
 
-#    - $TRAVIS_BUILD_DIR/truffle/node_modules
+    - node_modules
```

Comment:
Can you explain what this has to do with travis build speed?

Reply:
In general can you elaborate how these PR changes improves speed, so it will be more clear?

Reply:
I'll add it to the description

Reply:
:+1: